### ISSUE TITLE
Invalid distribution on Maven Central

### ISSUE THREAD

**Issue by ben-manes** [FastFilter/fastfilter_java#26]
Currently I am using 1.0 via jitpack.io due to using prior to a release and wanted to switch to Maven Central. Unfortunately that dependency cannot be resolved because the pom and file locations do not match. The [central dependency](https://search.maven.org/artifact/io.github.fastfilter/fastfilter/1.0/jar) should be resolvable as `io.github.fastfilter:fastfilter:1.0` but fails with,

```
> Could not resolve all files for configuration ':simulator:compileClasspath'.
   > Could not resolve io.github.fastfilter:fastfilter:1.0.
     Required by:
         project :simulator
      > Could not resolve io.github.fastfilter:fastfilter:1.0.
         > Could not parse POM https://repo.maven.apache.org/maven2/io/github/fastfilter/fastfilter/1.0/fastfilter-1.0.pom
            > Could not find io.github.fastfilter:fastfilter_java:1.0.
              Searched in the following locations:
                - https://repo.maven.apache.org/maven2/io/github/fastfilter/fastfilter_java/1.0/fastfilter_java-1.0.pom
              If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
```

The `fastfilter_java` artifactId cannot be resolved, which appears in the parent pom.

```xml
<parent>
  <groupId>io.github.fastfilter</groupId>
  <artifactId>fastfilter_java</artifactId>
  <version>1.0</version>
</parent>
<modelVersion>4.0.0</modelVersion>
<artifactId>fastfilter</artifactId>
<packaging>jar</packaging>
```

A minimal example demonstrating the failure is,
```xml
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
  <modelVersion>4.0.0</modelVersion>
  <groupId>com.example</groupId>
  <artifactId>fastfilter</artifactId>
  <version>1.0-SNAPSHOT</version>
  <dependencies>
    <dependency>
      <groupId>io.github.fastfilter</groupId>
      <artifactId>fastfilter</artifactId>
      <version>1.0</version>
    </dependency>
  </dependencies>
</project>
```

```console
[INFO] Scanning for projects...
[INFO]
[INFO] -----------------------< com.example:fastfilter >-----------------------
[INFO] Building fastfilter 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.162 s
[INFO] Finished at: 2021-06-18T20:18:38-07:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project fastfilter: Could not resolve dependencies for project 
com.example:fastfilter:jar:1.0-SNAPSHOT: Failed to collect dependencies at 
io.github.fastfilter:fastfilter:jar:1.0: Failed to read artifact descriptor for
io.github.fastfilter:fastfilter:jar:1.0: Failure to find io.github.fastfilter:fastfilter_java:pom:1.0 in 
https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted
until the update interval of central has elapsed or updates are forced -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

**Comment by thomasmueller** [comment]
Hi Ben,

I'm sorry, that is unfortunate... I will need to release the parent as well... I was hoping this is not necessary, but looks like it is. I will try next week.

Regards,
Thomas

**Comment by thomasmueller** [comment]
I fixed the issue (at least I hope) and deployed version 1.0.2. It may take a few hours until shows up on https://s01.oss.sonatype.org/content/repositories/releases/io/github/fastfilter/ 
Let's then see if that works better!

**Comment by thomasmueller** [comment]
Should be fixed with version 1.0.2.
